lib/core-integration/src/Test/Integration/Scenario/API/Shelley/StakePools.hs

```diff
@@ -417,58 +429,80 @@ spec = do
             ]
 
     it "STAKE_POOLS_LIST_01 - List stake pools" $ \ctx -> do
-        eventually "Listing stake pools shows expected information" $ do
```

Comment:
I think there's a _theoretical_ chance that the pool registration certificates haven't been discovered yet, but I'm not sure it's possible in practice.

It fails quicker without the default delay.

I guess we could use `eventuallyUsingDelay (5 * second)` also.